Kickstarter-iOS/Features/Comments/Controllers/CommentRepliesViewController.swift

```diff
@@ -189,6 +200,33 @@ extension CommentRepliesViewController: CommentComposerViewDelegate {
   }
 }
 
+// MARK: - CommentCellDelegate
+
+extension CommentRepliesViewController: CommentCellDelegate {
+  func commentCellDidTapHeader(author _: Comment.Author) {
+    guard AppEnvironment.current.currentUser != nil, featureBlockUsersEnabled() else { return }
+
+    if #available(iOS 15.0, *) {
+      if let sheet = self.view.filter({ $0.tag == actionSheetTag }) {
+        sheet.removeFromSuperview()
```

Comment:
Is this to prevent two sheets from being shown at the same time? If so, I would expect that we would track that state in another way, instead of in tags/the view hierarchy.

Reply:
See my comments on using `present` and `UIAlertController` instead - that would be the pattern I would recommend here, unless there's a constraint I'm missing in the designs that makes a custom approach necessary.

Reply:
Yeah, that was my goal here. I didn't see the view being removed from the view hierarchy when dismissing it and wanted to make sure it wasn't being left behind. How would you recommend tracking that state instead?

I'll address your `present` note in your other comment 😄